config.tfvars

```diff
@@ -77,6 +77,16 @@ jira_db_instance_class       = "db.t3.micro"
 jira_db_allocated_storage    = 100
 jira_db_iops                 = 1000
 
+# If you want to restore the database from a snapshot, uncomment the following line and provide the snapshot identifier. This will
```

Comment:
I think we should include a new "Section" for the following three parameters with it's own comment.

Something like:

```
####### Dataset restore
To restore the dataset into the newly created instance, uncomment the following lines and provide all necessary parameters. Please follow XY documentation to see the requirements for dataset restoration.
```